lib/cursor.mongodb.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
-const util = require('util');
+const api = {};
+api.util = require('util');
+
 const transformations = require('./transformations.js');
```

Comment:
Is it intentional that some required modules are put into `api` and some aren't?